components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponent.java

```diff
@@ -834,7 +835,35 @@ protected void unsetOAuth2RevocationProcessor(OAuth2RevocationProcessor oAuth2Re
         if (log.isDebugEnabled()) {
             log.debug("Unset Oauth2 revocation processor.");
         }
-        OAuth2ServiceComponentHolder.setOrganizationUserResidentResolverService(null);
+        OAuth2ServiceComponentHolder.getInstance().setRevocationProcessor(null);
+    }
+
+    @Reference(
+            name = "oauth2.token.persistence.handler",
```

Comment:
Here the name is persistence handle but we are registering a token revocation handler.
the two wordings server very different purposes.

Reply:
removed with with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2179/commits/69f8c55fb76f15cbfe22b3c551b17491dc1796b0